Release Doc updated with SCP-4.8.24 and SCP-B2B-1.0.21 (#894)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/changelog.mdx

**Storefront Compatibility Package** 
************************************/}

      <ChangelogEntry
          date="2026-05-20"
          title="Storefront Compatibility Package v4.8.24"
          components={['Storefront Compatibility Package']}
      >
          The Storefront Compatibility Package has been updated to include the following changes:

>           - **Login as Customer**: Resolved an issue with the backend popup for one-time login code generation.

Reviewer: ```suggestion
          - **Login as Customer**: Fixed a backend popup issue with one-time login code generation for Login as Customer.
```

**Storefront Compatibility B2B Package**
************************************/}

      <ChangelogEntry
          date="2026-05-20"
          title="Storefront Compatibility B2B Package v1.0.21"
          components={['Storefront Compatibility B2B Package']}
      >
          The Storefront Compatibility B2B Package has been updated to include the following changes:

>           - **Company Hierarchy**: Resolved an issue where the Company Hierarchy GraphQL returned an empty hierarchy after removing a child company when the admin was not assigned to all child companies.

Reviewer: ```suggestion
          - **Company Hierarchy**: Fixed a GraphQL issue where removing a child company returned an empty hierarchy when the admin lacked access to all child companies.
```
